### ISSUE TITLE
Run code through a web accessibility website

### ISSUE THREAD

**Issue by henryjr1** [henryjr1/deeplyrooted_msu#26]
The library wants to meet the WCAG AA standard for accessibility for example, individuals who are colorblind, poor vision, etc. should be able to still use our site to the fullest. Here are some tools to help check for issues on websites. Just document any issues you find and message me in discord.

aXe: https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US - Scans a page and reports issues

Wave: https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US - Scans a page for issues

WebAIM Contrast Checker: https://webaim.org/resources/contrastchecker/ - Let's you input color values and tells you how much contrast there is and whether it meets one of the WCAG standards

Google Lighthouse - https://developers.google.com/web/tools/lighthouse/#get-started - Runs many kinds of tests, but A11y is a test you can run. (Built into chrome now)

A11ycasts - https://www.youtube.com/watch?v=1uyQdC3LqLo&list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g - Google's you tube show about Accessibility 
(I would only worry about this one if anyone has any one wants to pursue web development)
